### PR TITLE
Bump d.py to 2.3.2

### PR DESCRIPTION
**Pull request by ChrisLovering**

### CODE REVIEW

File: docs/changelog.rst [hunk 1]
```diff
@@ -5,6 +5,10 @@ Changelog
 =========
 
 
+- :release:`10.2.0 <21th August 2023>`
```
Comment (shtlrs) [inline docs/changelog.rst:8]: This should be updated

**Review by shtlrs — APPROVED** [review]
LGTM, only the date needs to be updated.

File: docs/changelog.rst (hunk 1 above)
Comment (ChrisLovering) [reply docs/changelog.rst]: https://github.com/python-discord/bot-core/compare/b84ee8bc0b0652e415b61303ee53ae49d09141c7..a91c4546c77d10bb921b8e807631d375989e2fb3

Comment (shtlrs) [reply docs/changelog.rst]: m0rge.

**Pull request merged**